config.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
     },
     "pokemonSpawnTime" : 900000 ,// 15 minutes in milliseconds,
     "limit" : 1000,
+    "token" : "hastobefilledwithsupersecrettoken", //please choose secret token and communicate it privately
```

Comment:
Can you please make this env variable dependent? Docker will build yes or yes and it's hard to intervene in the config.json programmatically inside the docker container